index.html

```diff
@@ -4,51 +4,58 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-
-    <title>tip calculator</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>Tip Calculator</title>
   </head>
   <body>
-    <h1>Tip Calculator</h1>
-
-    <h3>bill Amount</h3>
-
-    <form name="myForm">
-      <div class="bill">
-        <input
-          type="number"
-          placeholder="Bill Total"
-          name="billTotal"
-          id="billTot"
-        />
+  <div id="container">
+
+      <h1>Tip Calculator</h1>
+      
+      <div id="calculator">
+      
```

Comment:
Indentations are all missing. `id=form` is inside `id=calculator`. Indent it once. Same thing for `h3` inside `form`, and so and for the rest of the children.